html/elements/link.json

```diff
@@ -82,35 +82,31 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": "25"
+                "version_added": "34"
               },
               "chrome_android": {
-                "version_added": true
+                "version_added": "mirror"
```

Comment:
"mirror" is to be used without "version_added", I think.

Reply:
The lint agrees :)